### PR TITLE
Change mentee application workflow

### LINKED ISSUE #340: Fix UX of the apply as a mentee page
**Is your feature request related to a problem? Please describe.**
A lot of users in the last program commented on the placement of the apply button and it not being that accessibility friendly 

**Describe the solution you'd like**
1. Analyse the current view and suggest a better workflow (could be something simple like moving the apply button to a different place)
2. Implement that suggestion

### PR DESCRIPTION
**Pull request by YaiyaSam**
## Purpose

The purpose of this PR is to fix #340

## Goals

Fix UX of the mentee application page

### CODE REVIEW

File: src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx [hunk 1]
```diff
@@ -106,6 +114,7 @@ function MenteeApplication() {
         });
       }
     } else {
+      console.log(values);
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx]: Let's remove this log

File: src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx [hunk 2]
```diff
@@ -140,18 +149,20 @@ function MenteeApplication() {
             form={form}
             layout="vertical"
             size="middle"
+            initialValues={mentor && { appliedMentor: mentor.id }}
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx]:
```suggestion
            initialValues={{ appliedMentor: mentor?.id }}
```

File: src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx [hunk 3]
```diff
@@ -140,18 +149,20 @@ function MenteeApplication() {
...
               rules={[
                 {
-                  required: true,
+                  required: !mentor,
                   message: 'Required',
                 },
               ]}
             >
               <Select
+                defaultValue={mentor && mentor.id}
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/RequestMentors/scenes/Application/index.tsx]:
```suggestion
                defaultValue={mentor?.id}
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Thank you! @YaiyaSam There are some minor changes requested.

**Comment by YaiyaSam** [comment]
> Thank you! @YaiyaSam There are some minor changes requested.

Done! Please review.

**Review by anjula-sack — APPROVED** [review]
Thank you! @YaiyaSam

**Pull request merged**